### ISSUE TITLE
bug(TABS): Tab panel content not accessible to multiple screen readers

### ISSUE THREAD

**Issue by bryan-sullivan-ascendlearning-com** [angular/components#21819]
The [WAI-ARIA Example of Tabs with Manual Activation](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html) puts a tabindex="0" on the tab panel to put it in the page tab sequence to facilitate movement to panel content for assistive technology users. 

Adding tabindex="0" to the mat-tab-body which has the role="tabpanel" attribute would bring the Angular material tabs more in line with the WAI-ARIA authoring practices and "facilitate movement to panel content for assistive technology users".

#### Reproduction

You can see the behavior in the official Stackblitz example: https://stackblitz.com/angular/mgokvrvonpe?file=src/app/tab-group-basic-example.ts

Steps to reproduce:
1. Load Stackblitz example
2. Start a screen reader: JAWS, NVDA or VoiceOver
3. Activate new tab using keyboard
4. Try to navigate to the tab panel so the screen reader can read the content 
 
#### Expected Behavior

A screen reader user should be able to navigate to the tab panel content so it can be read aloud by the screen reader.

#### Actual Behavior

NVDA and VoiceOver are not able to read the tab panel content. 

JAWS provides instructions to use JAWS key + ALT + M to switch to the controlled element, which allows the user to read the tab panel content. But NVDA and VoiceOver don't appear to provide a similar keyboard shortcut.

#### Environment

- Angular: All
- CDK/Material: All
- Browser(s): All
- Operating System (e.g. Windows, macOS, Ubuntu): All

**Comment by crisbeto** [comment]
There was a change to add this, but we decided not to. See the discussion on https://github.com/angular/components/pull/14808.

**Comment by bryan-sullivan-ascendlearning-com** [comment]
That discussion says they would document how developers can handle the issue themselves. Is the comment that mentions the documentation the result of said documentation? Or are there plans to add documentation to the overview on material.angular.io? An example would be nice considering it's a known, breaking issue for multiple screen readers. But I'd settle for a short blurb to warn developers on the overview that the default component isn't 100% accessible out of the box.

**Comment by crisbeto** [comment]
I think we may end up having to add a new API to the tab group since there's no access to the `mat-tab-body` so that you can set the `tabindex` yourself. In theory something like this should work, but it's not a great developer experience.

```
<mat-tab-group>
  <mat-tab label="First">
    <div tabindex="0">
      Content
    </div>
  </mat-tab>
</mat-tab-group>
```

**Comment by bryan-sullivan-ascendlearning-com** [comment]
That's exactly what I'm using as a quick fix. Thanks for the update.
